packages/cli/src/ceramic-daemon.ts

```diff
@@ -276,15 +276,18 @@ export class CeramicDaemon {
     )
 
     const [modules, params] = Ceramic._processConfig(ipfs, ceramicConfig)
-    modules.loggerProvider
-      .getDiagnosticsLogger()
-      .imp(
-        `Starting Ceramic Daemon at version ${packageJson.version} with config: \n${JSON.stringify(
-          opts,
-          null,
-          2
-        )}`
-      )
+    const diagnosticsLogger = modules.loggerProvider.getDiagnosticsLogger()
+    diagnosticsLogger.imp(
+      `Starting Ceramic Daemon at version ${packageJson.version} with config: \n${JSON.stringify(
+        opts,
+        null,
+        2
+      )}`
+    )
+    const ipfsId = await ipfs.id()
```

Comment:
any reason you put this here instead of in `core`?  Seems like it would fit well in `Ceramic._init()`, or even in IpfsTopology.

Reply:
The message is useful if you run Ceramic daemon on cloud. It gives you info on how to expose an IPFS node without fiddling with IPFS API. The daemon code is responsible for building IPFS connection. Let it handle exposing relevant diagnostic information as well.

IpfsTopology is about connecting the IPFS node to other nodes, so not quite a fit.
Core - it is responsible for too many things at once.

In some sense, it is an intersection of a single responsibility principle and Edge vs Kernel dichotomy.